projects/plugins/jetpack/extensions/plugins/payments/index.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * External dependencies
+ */
+import { ExternalLink } from '@wordpress/components';
+import { PluginPostPublishPanel } from '@wordpress/edit-post';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import analytics from '../../../_inc/client/lib/analytics';
+import useAutosaveAndRedirect from '../../shared/use-autosave-and-redirect/index';
+
+const PaymentsPostPublish = () => {
+	const paymentInfoUrl = 'https://wordpress.com/payments-donations/';
+	const { autosaveAndRedirect } = useAutosaveAndRedirect( paymentInfoUrl );
+	const trackClick = event => {
+		event.preventDefault();
+		analytics.tracks.recordEvent( 'jetpack_editor_payments_post_publish_click' );
+		autosaveAndRedirect( event );
```

Comment:
Apparently this doesn't open into `_blank`.
The hook has the logic for it already, but it's set up to only do so if `isWidgetEditor`:

https://github.com/Automattic/jetpack/blob/6e8a306f0bb3e8ce01437592c78ef4a462bde785/projects/plugins/jetpack/extensions/shared/use-autosave-and-redirect/index.js#L12

https://github.com/Automattic/jetpack/blob/6e8a306f0bb3e8ce01437592c78ef4a462bde785/projects/plugins/jetpack/extensions/shared/use-autosave-and-redirect/index.js#L91

IMHO this is not a big enough deal to block this PR.
We can update the `useAutosaveAndRedirect` hook (or the `autosaveAndRedirect`) to also accept a `shouldOpenNewWindow` option, but it makes more sense to do it in a separate PR.

Reply:
Good point. I just committed b2cf36f67fd621d0651e9aead121b4ea5a4e3677